test-case/check-kmod-load-unload-after-playback.sh

```diff
@@ -55,33 +55,22 @@ pb_duration=${OPT_VALUE_lst['d']}
 func_pipeline_export "$tplg" "type:playback"
 
 func_lib_check_sudo
-# overwrite the subscript: test-case LOG_ROOT environment
-# so when loading the test-case in current script
-# the test-case will write the log to the store folder LOG_ROOT
-# which is the current script log folder
-log_root=$LOG_ROOT
 
 if [ ${OPT_VALUE_lst['p']} -eq 1 ];then
     func_lib_disable_pulseaudio
 fi
 
-export LOG_ROOT=$log_root/round-0
-$(dirname ${BASH_SOURCE[0]})/check-playback.sh -l 1 -t $tplg -d $pb_duration
-ret=$?
-export LOG_ROOT=$log_root
-[[ $ret -ne 0 ]] && dloge "aplay check failed" && exit $ret
+$(dirname ${BASH_SOURCE[0]})/check-playback.sh -l 1 -t $tplg -d $pb_duration ||
```

Comment:
While you're changing these lines, compute this once at the top of the file.
```shell
mydir=$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd)
```

Reply:
The feature is to generate logs of each iteration to different folder, but our CI is not aware of these `round-X` folder, so logs cannot show in report. And I am sure no one is really aware of the logs in `round-X` folder.

I will update `mydir`, thanks